pkg/render/node.go

```diff
@@ -1021,6 +1022,7 @@ func (c *nodeComponent) nodeEnvVars() []v1.EnvVar {
 		{Name: "WAIT_FOR_DATASTORE", Value: "true"},
 		{Name: "CLUSTER_TYPE", Value: clusterType},
 		{Name: "CALICO_DISABLE_FILE_LOGGING", Value: "true"},
+		{Name: "CALICO_MANAGE_CNI", Value: "true"},
```

Comment:
Do we need to handle clusters migrating from where `CALICO_MANAGE_CNI` is false?
I guess if the Calico CNI monitor-token loop was running, it would could just run and not do anything.

Reply:
yeah, migration should be fine and shouldn't have an impact.